packages/x-date-pickers/src/internals/hooks/useConvertFieldResponseIntoMuiTextFieldProps.ts

```diff
@@ -0,0 +1,17 @@
+import { TextFieldProps } from '@mui/material/TextField';
+import { UseFieldResponse } from './useField';
+
+export const useConvertFieldResponseIntoMuiTextFieldProps = <
```

Comment:
With #10649 this will convert either to the `MuiTextField` or `PickersTextField` format.